<launch>

    <node ns="yk_architect" name="enable_robot" pkg="yk_interface" type="rosservice_enable.py" output="screen">
		<param name="namespace" type="str" value="yk_architect" />
	</node> 

    <node ns="yk_builder" name="enable_robot" pkg="yk_interface" type="rosservice_enable.py" output="screen">
		<param name="namespace" type="str" value="yk_builder" />
	</node> 

    <node ns="yk_creator" name="enable_robot" pkg="yk_interface" type="rosservice_enable.py" output="screen">
		<param name="namespace" type="str" value="yk_creator" />
	</node> 

    <node ns="yk_destroyer" name="enable_robot" pkg="yk_interface" type="rosservice_enable.py" output="screen">
		<param name="namespace" type="str" value="yk_destroyer" />
	</node> 

</launch>
